src/app/login/login.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit } from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+  rForm: FormGroup;
+  post: any;                     // A property for our submitted form
+  password: string = '';
+  name: string = '';
+  titleAlert: string = 'This field is required';
```

Comment:
change variable name to `fieldRequiredAlert`. `titleAlert` doesnt make sense

Reply:
changed